vpp.spec

```diff
@@ -1,4 +1 @@
-VPP_COMMIT=3d2d96e5547484290c9368bac0a420afa8c4c068
-VPP_REPO=https://github.com/fdio/vpp.git
-VPP_BRANCH=stable/2101
-VPP_RELEASE=v21.01
+VPP_IMAGE_BASE=registry.cennso.com/upg/vpp-base:21.01.0-21-ba7f5d10f
```

Comment:
Before merging this PR, we need to update vpp-base image to official released one.

Reply:
Right now the situation with NAT patch is a bit indefinite, let's see if we make 100% sure the NAT patch is final first, or maybe we can merge the "great split" PRs and update later